packages/cli/src/util/events.ts

```diff
@@ -10,6 +10,7 @@ import Client from './client';
 import getDeployment from './get-deployment';
 import getScope from './get-scope';
 
+import { BuildLog } from './logs';
```

Comment:
**suggestion:** please import it as a type.

I'll be surprised if eslint doesn't rant about it <picture data-single-emoji=":lol-think:" title=":lol-think:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/lol-think/d89d6dcd2efc2a05.png" alt=":lol-think:" align="absmiddle"></picture>

Reply:
surprised you shall be XD